app/settings/prestart.sh

```diff
@@ -1,2 +1,17 @@
 #!/usr/bin/env bash
+if [ "${ENV}" = "dev" ]; then
```

Comment:
Could you put some comments on what you're doing here for each of these and why? All the bash commands are a little dense.